enterprise/payara.micro/nbproject/org-netbeans-modules-payara-micro.sig

```diff
@@ -40,7 +40,6 @@ fld public final static java.lang.String PROP_ARTIFACT_ID = "artifactId"
 fld public final static java.lang.String PROP_AUTO_BIND_HTTP = "autoBindHttp"
 fld public final static java.lang.String PROP_CONTEXT_ROOT = "contextRoot"
 fld public final static java.lang.String PROP_GROUP_ID = "groupId"
-fld public final static java.lang.String PROP_JAVA_EE_VERSION = "javaeeVersion"
```

Comment:
Removing a field from an API is probably completely unnecessary. Rather mark as `@Deprecated`.